meta/runtime.yml

```diff
@@ -541,6 +541,11 @@ plugin_routing:
       deprecation:
         removal_version: 3.0.0
         warning_text: see plugin documentation for details
+    firewalld:
+      deprecation:
+        removal_version: 2.0.0
+        warning_text: migrated to the ansible.posix collection
+      redirect: ansible.posix.firewalld
```

Comment:
Sorry for not realizing this earlier: you need a similar entry for the `firewalld` module_utils (in the `plugin_routing.module_utils` dict).

Reply:
no worries, thanks for catching that ... I 100% did not even notice I missed that :+1: